tools/proofers/text_frame_std.inc

```diff
@@ -3,7 +3,7 @@ include_once($relPath.'button_defs.inc');
 include_once($relPath.'slim_header.inc');
 include_once($relPath.'misc.inc');
 include_once('preview.inc');
-include_once('codepoint_validator.inc');
+include_once($relPath.'codepoint_validator.inc');
```

Comment:
Move this up two lines before the `preview.inc` with the others that pull from `pinc/`

Reply:
Done this.